sql/core/src/test/scala/org/apache/spark/sql/StreamTest.scala

```diff
@@ -280,19 +283,35 @@ trait StreamTest extends QueryTest with Timeouts {
     try {
       startedTest.foreach { action =>
         action match {
-          case StartStream =>
+          case StartStream(_trigger, _triggerClock) =>
             verify(currentStream == null, "stream already running")
             lastStream = currentStream
             currentStream =
-              sqlContext
-                .streams
-                .startQuery(
-                  StreamExecution.nextName,
-                  metadataRoot,
-                  stream,
-                  sink,
-                  outputMode = outputMode)
-                .asInstanceOf[StreamExecution]
+              if (_trigger != null) {
```

Comment:
This is minor, but why the `_underscores` here?  Its also a little confusing that we have these layers of default arguments.  Should we get rid of the defaults in `startQuery` and only have them in this test code?